src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSenderTest.java

```diff
@@ -86,8 +100,8 @@ void should_handle_no_attachments() throws Exception {
             emptyMap()
         );
 
-        MimeMessageParser msg = new MimeMessageParser(greenMail.getReceivedMessages()[0]).parse();
-
+        assertThat(smtpServer.getReceivedMessages()).hasSize(RECIPIENTS.length);
+        MimeMessageParser msg = new MimeMessageParser(smtpServer.getReceivedMessages()[0]).parse();
```

Comment:
Not sure this is the correct way but I found something on below lines. 

`smtpServer.getReceivedMessages()` will contains headers , body so extracting using util will make it clear imo.
```
  GreenMail mailServer = TestUtil.TestRuntimeModule.getMailServer();
    while(mailServer.getReceivedMessages().length < 1) {
      ThreadUtil.sleep(100);
    }
    String headers = GreenMailUtil.getHeaders(mailServer.getReceivedMessages()[0]);
    Assert.assertTrue(headers.contains("To: foo, bar"));
    Assert.assertTrue(headers.contains("Subject: StreamsSets Data Collector Alert - " +
        TestUtil.PIPELINE_TITLE_WITH_EMAIL + " - STOPPED"));
    Assert.assertTrue(headers.contains("From: sdc@localhost"));
    Assert.assertNotNull(GreenMailUtil.getBody(mailServer.getReceivedMessages()[0]));

```